PlayFabClientSDK/source/PlayFabHttp/IPlayFabHttp.cs

```diff
@@ -1,10 +1,9 @@
-using System.Collections.Generic;
-using System.Threading.Tasks;
+using System;
 
 namespace PlayFab.Internal
 {
-    public interface IPlayFabHttp
+    [Obsolete("This interface is deprecated, please use PlayFab.ITransportPlugin instead.", false)]
```

Comment:
Can you explain why we are keeping the interface if it is obsolete? Is it just backwards compatibility for folks who download a new version?

Reply:
Yes, just for that. There is no other reason to keep it around.

---
In reply to: [201477286](https://github.com/PlayFab/CSharpSDK/pull/97#discussion_r201477286) [](ancestors = 201477286)